windows/windowsservercore/Dockerfile

```
@@ -23,10 +23,11 @@
 #  THE SOFTWARE.
 
 ARG JAVA_VERSION=17.0.7_7
-ARG WINDOWS_VERSION_TAG=1809
-FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-windowsservercore-"${WINDOWS_VERSION_TAG}" AS jdk-core
+ARG WINDOWS_VERSION_TAG=ltsc2019
+ARG WINDOWS_VERSION_FALLBACK_TAG=1809
+FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-windowsservercore-"${WINDOWS_VERSION_FALLBACK_TAG}" AS jdk-core
```

```suggestion
FROM eclipse-temurin:"${JAVA_VERSION}"-jdk-windowsservercore-"${WINDOWS_VERSION_FALLBACK_TAG}" AS jdk-core
```

nitpicking on naming: `WINDOWS_VERSION_FALLBACK_TAG` feels hard to get. 

WDYT about `TOOLS_WINDOWS_VERSION` instead?

- prefix `TOOLS` to make it clear for what it is used (temurin and pwsh)
- no more `TAG` suffix because it is not a tag (neither Docker image tag nor git tag)